BlinkID/example/lib/document_scanner.dart

```diff
@@ -0,0 +1,173 @@
+import 'package:blinkid_flutter/microblink_scanner.dart';
+import 'package:blinkid_flutter/microblink_scanner_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:permission_handler/permission_handler.dart';
+
+typedef MicroblinkScannerResultCallback = Future<bool> Function(List<RecognizerResult>?);
```

Comment:
Any `typedef` is a detail. Details must not be at the top of the source file.

Also, this particular type should be private.

But ultimately, delete this, cause there is the same stuff coming from `import 'package:blinkid_flutter/microblink_scanner_widget.dart';`